src/igz_controls/components/more-info/more-info.tpl.html

```diff
@@ -0,0 +1,19 @@
+<div class="more-info-wrapper">
+    <div data-ng-if="!$ctrl.isDefaultTooltipEnabled"
+         data-ng-class="{'disabled': $ctrl.isDisabled}"
+         class="question-mark igz-icon-help-round">
+    </div>
+    <div data-ng-if="!$ctrl.isDefaultTooltipEnabled"
+         class="row-description wizard-row-description">
```

Comment:
Hmm… So here you rely on CSS classes being defined in the parent component that would use `<igz-more-info>`? 
`wizard-row-description` is not defined in this (dashboard-controls) repo.

Reply:
Okay, I see that now `wizard-row-description` is redundant and we can remove it. And `row-description` defining has been moved into `more-info.less` file.

Reply:
It is better now, but the `row-description` CSS class is used more than 100 times in Dashboard repo in wizards and dialogs.
Is `igzMoreInfo` dependent on that?
I see you use it in the component's HTML template here: https://github.com/iguazio/dashboard-controls/pull/555/files#diff-3f22e087d0dda8a4c8c76cf317ebafffR7
If you rename this CSS class to something else, will it still work properly on Dashboard?

Reply:
If I rename this class it will work properly on Dashboard, because this class doesn't use on Dashboard anymore, only in `igzMoreInfo`. I've removed all places where it was used for positioning tooltip.

Reply:
Right, I'm sorry, I checked this on development branch instead of on this PR's branch. Now I see you removed it. Good.